jobfunnel/config/parser.py

```diff
@@ -5,21 +5,39 @@
 import logging
 import os
 import yaml
+from jobfunnel.tools import easy_apply
 
 log_levels = {'critical': logging.CRITICAL, 'error': logging.ERROR,
               'warning': logging.WARNING, 'info': logging.INFO,
               'debug': logging.DEBUG, 'notset': logging.NOTSET}
 
 
+def _parse_easy_apply(parser):
```

Comment:
I think these argparse commands make more sense in the existing _parse_cli method